apps/epic-web/src/pages/buy.tsx

```diff
@@ -86,6 +89,55 @@ const BuyPage: React.FC<
         <h1 className="max-w-lg text-2xl font-bold sm:text-3xl lg:text-4xl">
           <Balancer>Become a Professional Full Stack Web Developer</Balancer>
         </h1>
+        <CourseJsonLd
+          courseName="Full Stack Vol 1"
+          description="The most comprehensive guide to professional web development by Kent C. Dodds."
+          provider={{
+            name: 'Kent C. Dodds',
+            type: 'Person',
+            url: isBrowser()
+              ? document.location.href
+              : process.env.NEXT_PUBLIC_URL,
+          }}
```

Comment:
`sameAs` is being passed by default which we can add kents socials here and personal site. This could be 

```json
{
  sameAs: ['https://kentcdodds.com', 'https://twitter.com/kentcdodds', 'https://github.com/kentcdodds'] 
}
```

https://schema.org/sameAs

![image](https://github.com/skillrecordings/products/assets/6188161/50edf1b1-e985-46d2-a6a6-6413aff605a2)

Reply:
Unfortunately, the provider prop doesn't support sameAs. I think that's the intended use of the URL prop. But, you can't pass an array to it. 

I'll set Kent's personal site as that URL.